src/main/java/svc/hashids/HashidsConfig.java

```diff
@@ -0,0 +1,30 @@
+package svc.hashids;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class HashidsConfig {
+	@Value("${stlcourts.security.municipalitySalt}")
```

Comment:
Can you simplify this into 1 bean that provides a new object of your creation (maybe "Hasher" or something). That class would have an encrypt/decrypt (or hash/unhash) function that would take 1) the class type and 2) the id. Then in that function, you can use a switch to determine which hashids object to use (which you would want to statically create in this "Hasher". It buries the logic and gives you a clean point of entry to all areas that need this functionality